### PR TITLE
DEVOPS-1805 Simply steps to run configurator.

### PR DESCRIPTION
**Pull request by bjhaid**
This relies on only the presence of a values.yaml in current working
directory and leverages the configs add to the image to generate
manifests. This technically cuts away 2 steps from the steps required to
run the configurator, since there is no need for the user to clone repo
and build the image themselves.

Anticipated usage:

`docker run -v ${PWD}:/manifests --rm -it ${img}`

Depends on:

#194

### CODE REVIEW

**Comment by bjhaid** [comment]
Only this commit:

https://github.com/draios/sysdigcloud-kubernetes/pull/195/commits/a6e355e0f0c77e48013681e6a602911deb65caed

belongs to this PR, the other is from 

https://github.com/draios/sysdigcloud-kubernetes/pull/194

which is yet to be merged

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,38 @@
+pipeline {
+  agent {
+    node { label 'builder-backend-j8' }
```
Comment (markyjackson-taulia) [inline Jenkinsfile]: Can you remove this and make it an env var? We can set these internally if need be

Comment (markyjackson-taulia) [reply Jenkinsfile]: You can make it `${WORKER}`

Comment (bjhaid) [reply Jenkinsfile]: Thanks addressed

**Pull request merged**